pages/finance/creating_personal_purchases.mdx

```diff
@@ -0,0 +1,9 @@
+# Creating Personal Purchases
+
+1. Go to https://finance-frontend.watonomous.ca/
+2. Create New Ticket > Ticket Type: Personal Purchase
+3. Enter all purchase details. For funding item link, you would either be using FI-1 (WATO Cash) or one of the Funding Items that match the category. For example, if you're purchasing an RTX 4090 it would probably fit under GPU funding.
+4. Await team approval. Please reach out to your team director, team captain, and the faculty advisor.
```

Comment:
does "team director, team captain, and faculty advisor" still make sense? would "finance team and faculty advisor" make more sense?